extensions/typescript-language-features/src/commands/allowPromptUseWorkspaceTsdk.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import TypeScriptServiceClientHost from '../typeScriptServiceClientHost';
+import { Command } from '../utils/commandManager';
+import { Lazy } from '../utils/lazy';
+
+export class AllowPromptUseWorkspaceTsdkCommand implements Command {
```

Comment:
Is this command being used?

Reply:
No, and I'd be happy to remove it. I felt weird not adding a way to reverse this internal state, but if there are other examples of non-reversible state than I don't really see why this would be needed ever